src/simularium/RemoteSimulator.ts

```diff
@@ -156,17 +156,17 @@ export class RemoteSimulator implements ISimulator {
 
         this.webSocketClient.addJsonMessageHandler(
             NetMessageEnum.ID_UPDATE_TIME_STEP,
-            this.updateTimestep
+            (_msg) => this.updateTimestep()
         );
 
         this.webSocketClient.addJsonMessageHandler(
             NetMessageEnum.ID_UPDATE_RATE_PARAM,
-            this.updateRateParam
+            (_msg) => this.updateRateParam()
         );
 
         this.webSocketClient.addJsonMessageHandler(
             NetMessageEnum.ID_MODEL_DEFINITION,
-            this.onModelDefinitionArrive
+            (_msg) => this.onModelDefinitionArrive()
```

Comment:
changes in this file are a bonus bug fix - the heartbeat ping was raising exceptions in react and breaking the viewer, not sure when bug was introduced.